configs/webpack.config.renderer.dev.js

```diff
@@ -61,6 +61,7 @@ export default merge.smart(baseConfig, {
     rules: [
       {
         test: /\.jsx?$/,
+        include: path.join(__dirname, 'app'),
```

Comment:
The path to `src` is not `path.join(__dirname, 'src')` -- it's `path.join(__dirname, '../src')`.